www/src/components/layer-model/layer-model.js

```diff
@@ -116,7 +85,17 @@ const LayerModel = ({ initialLayer = `Content` }) => {
     }
   }, [selected])
   return (
-    <>
+    <div 
+      sx={{
+        borderRadius: 3,
+        border: t => `1px solid ${t.colors.pullquote.borderColor}`,
```

Comment:
```suggestion
        border: t => `1px solid ${t.colors.ui.border}`,
```

IMO works equally well:

![image](https://user-images.githubusercontent.com/21834/76251172-e8baa400-6246-11ea-85fd-7235c921e9cb.png)

![image](https://user-images.githubusercontent.com/21834/76251211-f708c000-6246-11ea-918b-03c58b43fdc8.png)

… although I would like a little more contrast, so still leaves some follow up work. ;)
Either is good to ship IMO.

Reply:
Ah, FWIW — I just noticed that the `shell` code block (containing `npm install gatsby-transformer-sharp gatsby-plugin-sharp gatsby-image`) is not having any padding on smaller screens; and I wondered about adding a "Copy" button for the `npm install` bit?

Reply:
Thanks for the comments @fk! I'll jump in and make some changes.

Reply:
Just updated, here are some screenshots:

![image](https://user-images.githubusercontent.com/21114044/76265165-74bdd300-6229-11ea-8725-3154a6872e0f.png)

![image](https://user-images.githubusercontent.com/21114044/76265189-81422b80-6229-11ea-8022-2ca5afbd0779.png)

![image](https://user-images.githubusercontent.com/21114044/76265230-9323ce80-6229-11ea-95d0-305c6ea7fe7e.png)

Reply:
Looks great Kyle!